activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -179,20 +179,9 @@ def prepare_all
 
         each_current_configuration(env) do |db_config|
           with_temporary_pool(db_config) do
```

Comment:
I notice `initialize_database` already has `with_temporary_pool`, so it looks like we're nesting temporary pools. Is it safe to remove it here?